asmcomp/cmm_helpers.mli

```diff
@@ -214,6 +214,10 @@ val set_field :
   expression -> int -> expression -> Lambda.initialization_or_assignment ->
   Debuginfo.t -> expression
 
+(** [field_address_computed ptr ofs dbg] returns an expression for the address
+    at offset [ofs] of the block pointed to by [ptr]. *)
+val field_address_computed : expression -> expression -> Debuginfo.t -> expression
```

Comment:
We don't support 32-bit architectures in the native compiler anymore, so it would actually work for flat float records too. But it would be nice to document that the offset is in machine words, and maybe that the resulting expression is a derived pointer (of type `Addr`).